HCTextField/Source/HCTextField.swift

```diff
@@ -163,6 +170,38 @@ open class HCTextField: UITextField {
             layer.borderColor = HCTextFieldBorderColor.error.cgColor
         }
     }
+
+    private func addCheckmark() {
+        rightViewMode = .unlessEditing
+
+        guard self.rightContainerView == nil else {
+            rightView = self.rightContainerView
+            return
+        }
+
+        let dividerWidth: CGFloat = 5
+        let imageViewWH = frame.size.height * 0.8
+        let containerViewWidth = imageViewWH + dividerWidth
+
+        let rightContainerView = UIView(frame: CGRect(x: 0, y: 0, width: containerViewWidth, height: imageViewWH))
+
+        let imageView = UIImageView(frame: CGRect(x: 0, y: 0, width: imageViewWH, height: imageViewWH))
+        imageView.image = UIImage(named: "checkmark")
```

Comment:
let the application pass the checkmark image to you. Add a optional property of checkmark.